main.js

```diff
@@ -0,0 +1,69 @@
+let bookCollection = [];
```

Comment:
Please fix it so that the remove book is working right now it is only working when I reload the page and it is not working on the page as I remove it according to requirements.